src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -144,7 +146,7 @@ public int hashCode()
 		int result = super.hashCode();
 		result = prime * result + Objects.hash(attributes, exp, expLevel, expTotal, displayName, gamemode, getHealth(),
 				foodLevel, saturation, inventory, enderChest, inventoryView, getMaxHealth(), online, whitelisted,
-				compassTarget, bedSpawnLocation, cursor);
+				compassTarget, bedSpawnLocation, cursor, firstPlayed, lastPlayed);
```

Comment:
The `PlayerMock#equals` method should also test for `firstPlayed` and `lastPlayed`

Reply:
Whoops, my bad.